packages/map/src/GMap.ts

```diff
@@ -553,6 +560,8 @@ export class GMap extends HTMLWidget {
                 });
                 const heading = google.maps.geometry.spherical.computeHeading(data.location.latLng, new google.maps.LatLng(pos.lat, pos.lng));
                 context._googleMapPanorama.setPano(data.location.pano);
+                // context._googleMapPanorama.setPano(data.time[0].pano);//TODO: use this data point to create map_GMapStreetViewTimelapse
+                
```

Comment:
Remove commented code.

Reply:
I left this here because of the "TODO". As it turns out... most (if not all) street view panoramas have several captures years apart. What do you think of the idea to create a streetview timelapse widget?

Reply:
Sure - but you can create a new branch with it in and then remove the commented code here (otherwise commented code has a habit of never going away).